File: gateOne/DayOne/Java Script codes/taskTen.js
let sum = 0;
let average = 0;
let counter = 0;

for(let count = 0;count < 10;count++) {
	const prompt = require ("prompt-sync")();
	let score = parseInt(prompt("Enter Number: "))
	if(score > 0){	
		sum += score;
		counter++;}
}
average = sum / counter;

console.log(sum);
console.log(average); 

File: gateOne/DayOne/Python codes/task_ten.py
sum = 0
average = 0
COUNTER = 0

for number in range(1,11,1):
	user_input = int(input("Enter Number \n"))
	if user_input > 0:
		sum += user_input
		COUNTER = COUNTER + 1

average = sum / counter		
print(average)



